client/src/Layout/DashboardLayout.tsx

```diff
@@ -22,17 +30,28 @@ function DashboardLayout({ component }: Props): JSX.Element {
 
   return (
     <Box className={classes.root}>
-      <AppBar position="relative" color="inherit" elevation={6}>
+      <AppBar position="sticky" color="inherit" elevation={6} className={clsx(isHomePage && classes.HomePage)}>
```

Comment:
We don't really need a state to keep track of the homepage, you can add a conditional as below:

Eg:
```
className={clsx({                  
          [classes.HomePage]: pathname === '/home', 
        })}
```